Configuration/DataProcessing/python/Merge.py

```diff
@@ -74,6 +74,8 @@ def mergeProcess(*inputFiles, **options):
         process.add_(Service("InitRootHandlers", EnableIMT = CfgTypes.untracked.bool(False)))
     else:
         outMod = OutputModule("PoolOutputModule")
+        outMod.mergeJob = CfgTypes.untracked.bool(True)
+        outMod.eventAuxiliaryBasketSize = CfgTypes.untracked.int32(2*1024*1024)
```

Comment:
> Also, the basket size for the EventAuxiliary branch is bumped in order to improve the data layout for sites with large block sizes.

Should we think of increasing this by default too? Or are the benefits more or less specific to outputs of merge jobs?

Reply:
I believe this is mostly specific to the merge jobs, we generally don't have single jobs producing 20GB files, and we don't fast-clone the EventAuxiliary branch, so it should be sufficient to do this just for merges.